reviewer-2-comments.md

```diff
@@ -0,0 +1,13 @@
+# Reviewer 2
```

Comment:
maybe add in some comments from reviewer 1, who doesn't recommend any changes, loves it 100%, and thinks anyone who thinks it needs to be changed is an idiot? idk, maybe just a 👍 lol. update filename to just reviewer-comments if you decide to do this.

Reply:
Yeah, I'm not as funny as I thought haha